realsense2_camera/CMakeLists.txt

```diff
@@ -93,7 +93,7 @@ endif()
 
 find_package(ament_cmake REQUIRED)
 find_package(builtin_interfaces REQUIRED)
-find_package(cv_bridge REQUIRED)
+find_package(cv_bridge REQUIRED 3.3.0)
```

Comment:
We shouldn't set a specific version. The type adapter is only available in rolling and hence this PR should also target only the rolling distro. The code base can internally use `#ifdef ROLLING` to decide whether to include the adapter headers and divert the logic of the code